docs/developer/README.md

```diff
@@ -64,7 +64,7 @@ this only works on a Linux host.
 ```
 # will create the cluster, and the agent config files in bin/agent which will be embedded in the image
 make deploy
-make agent-vm agent-vm-console # user/password is user/user
+make deploy-e2e-extras agent-vm agent-vm-console # user/password is user/user
```

Comment:
thanks for doing this PR I just wonder if the registry should just be part of deploy. as a dev this is an extra step we need to do fairly often.

Reply:
I guess that makes sense. wdyt @mangelajo ?

Reply:
make it a dependency to deploy? that would make sense I think, I don't remember now if there was any technical reason why I didn't add it to deploy. I know I split it from the deploy script to be able to do it to a target cluster instead (OpenShift),

https://github.com/flightctl/flightctl/blob/main/deploy/deploy.mk#L9

We could add it here right before the prepare-agent

Reply:
Updated!